### DISCUSSION TITLE
Initial Password Limitations

### DISCUSSION THREAD

**Discussion by gcsrellis** [Security-Onion-Solutions/securityonion#1989] [Unsupported Versions] [answered]
Is there a password limitation on initial installation?  I created a username and a password that was 17 characters, uppercases, and special characters.  When it finishes the install and reboots it will not accept my password.  I restarted the install with a 5 character password and it works.

**Comment by dougburks** [comment, marked as answer]
It's most likely the special characters. This should be improved in the next release:
https://github.com/Security-Onion-Solutions/securityonion/issues/1639
